crates/core/src/commands/check.rs

```diff
@@ -676,7 +707,17 @@ fn check_pack(
     // check header
     let header = be.decrypt(&data.split_off(data.len() - header_len as usize))?;
 
-    let pack_blobs = PackHeader::from_binary(&header)?.into_blobs();
+    let pack_blobs = PackHeader::from_binary(&header)
+        .map_err(|err| {
+            RusticError::with_source(
+                ErrorKind::Command,
```

Comment:
Same: Probably `Internal` is better here. Not sure, if it even makes sense to have the `Command` variant. As most of that is also internal.